rustfmt.toml

```diff
@@ -0,0 +1 @@
+
```

Comment:
if we don't config anything, why do we have this file even

Reply:
I'm going to be extremely annoying and suggest a config that matches what I was trying to do manually

```
[formatting]
align_comments = true
indent_string = "    "
reorder_arrays = true
reorder_inline_tables = true
reorder_keys = true
```

Reply:
🤷 pr to demo what these look like practically ;)

Reply:
https://github.com/gfx-rs/wgpu/pull/6376

Reply:
@cwfitzgerald: I've moved the addition of a `taplo.toml` with the indentation change into this PR. This minimizes diffs. in this PR and #6376. This movement also narrows the focus of that PR on setting `reorder_*` config. to `true`, which I think is a nice, contained change that we can merge after this. 🙂